.byebug_history

```diff
@@ -0,0 +1,54 @@
+exit
+subject.deadable?
+subject.dead?
+subject.deadable
+subject
+worker.deadable
```

Comment:
Hi @mathieujobin ! Thanks for your advice! You are right. I just wanted push some changes in order to have a second look from a work colleague and by mistake I added the commit on this branch.